daemon/info.go

```diff
@@ -174,7 +175,13 @@ func (daemon *Daemon) showPluginsInfo() types.PluginsInfo {
 
 	pluginsInfo.Volume = volumedrivers.GetDriverList()
 	pluginsInfo.Network = daemon.GetNetworkDriverList()
-	pluginsInfo.Authorization = daemon.configStore.AuthorizationPlugins
+
+	authPlugins := []string{}
+	for _, p := range daemon.configStore.AuthorizationPlugins {
+		authPlugins = append(authPlugins, p)
+	}
+	sort.Strings(authPlugins)
+	pluginsInfo.Authorization = authPlugins
```

Comment:
@allencloud would prefer to have this in a function like `daemon.GetNetworkDriverList()` or `volumedrivers.GetDriverList()`